drivers/mrf24j40/mrf24j40_getset.c

```diff
@@ -0,0 +1,594 @@
+/*
+ * Copyright (C) 2016 Hochschule f�r Angewandte Wissenschaften
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mrf24j40
+ * @{
+ *
+ * @file
+ * @brief       Getter and setter functions for the MRF24J40 drivers
+ *
+ * @author      Tobias Fredersdorf <tobias.fredersdorf@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include "mrf24j40.h"
+#include "mrf24j40_internal.h"
+#include "mrf24j40_registers.h"
+#include "periph/spi.h"
+#include "xtimer.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+/* Values of RFCON3 - Address: 0x203
+ * 0b00000000 -> 0dB	-> 0
+ * 0b00001000 -> -0.5dB -> 0
+ * 0b00010000 -> -1.2dB -> -1
+ * 0b00011000 -> -1.9dB -> -2
+ * 0b00100000 -> -2.8dB -> -3
+ * 0b00101000 -> -3.7dB -> -4
+ * 0b00110000 -> -4.9dB -> -5
+ * 0b00111000 -> -6.3dB -> -6
+ * 0b01000000 -> -10dB  -> -10
+ * 0b01001000 -> -10.5dB-> -10
+ * 0b01010000 -> -11.2dB-> -11
+ * 0b01011000 -> -11.9dB-> -12
+ * 0b01100000 -> -12.8dB-> -13
+ * 0b01101000 -> -13.7dB-> -14
+ * 0b01110000 -> -14.9dB-> -15
+ * 0b01111000 -> -16.3dB-> -16
+ * 0b10000000 -> -20dB  -> -20
+ * 0b10001000 -> -20.5dB-> -20
+ * 0b10010000 -> -21.2dB-> -21
+ * 0b10011000 -> -21.9dB-> -22
+ * 0b10100000 -> -22.8dB-> -23
+ * 0b10101000 -> -23.7dB-> -24
+ * 0b10110000 -> -24.9dB-> -25
+ * 0b10111000 -> -26.3dB-> -26
+ * 0b11000000 -> -30dB  -> -30
+ * 0b11001000 -> -30.5dB-> -30
+ * 0b11010000 -> -31.2dB-> -31
+ * 0b11011000 -> -31.9dB-> -32
+ * 0b11100000 -> -32.8dB-> -33
+ * 0b11101000 -> -33.7dB-> -34
+ * 0b11110000 -> -34.9dB-> -35
+ * 0b11111000 -> -36.3dB-> -36
+ */
+
+static const int16_t tx_pow_to_dbm[] = { 0, 0, -1, -2, -3, -4, -5, -6,
+                                         -10, -10, -11, -12, -13, -14, -15, -16,
+                                         -20, -20, -21, -22, -23, -24, -25, -26,
+                                         -30, -30, -31, -32, -33, -34, -35, -36 };
+
+static const uint8_t dbm_to_tx_pow[] = { 0xf8, 0xf0, 0xe8, 0xe0, 0xd8, 0xd0, 0xc8, 0xc0,
+                                         0xb8, 0xb0, 0xa8, 0xa8, 0x98, 0x90, 0x88, 0x80,
+                                         0x78, 0x70, 0x68, 0x60, 0x58, 0x50, 0x48, 0x40,
+                                         0x38, 0x30, 0x28, 0x20, 0x18, 0x10, 0x08, 0x00 };
+
+

```

Comment:
Hello miri64, 
thanks a lot for the hint - I 'll make a correction tomorrow.

Regards,
Neo